src/avx512-common-qsort.h

```diff
@@ -405,6 +389,46 @@ X86_SIMD_SORT_INLINE arrsize_t partition_avx512_unrolled(type_t *arr,
     reg_t min_vec = vtype::set1(*smallest);
     reg_t max_vec = vtype::set1(*biggest);
 
+    int64_t vecsToPartition = ((right - left) / vtype::numlanes) % num_unroll;
+    type_t buffer[num_unroll * vtype::numlanes];
+    int32_t bufferStored = 0;
+    int64_t leftStore = left;
+
+    for (int32_t i = 0; i < vecsToPartition; i++) {
+        reg_t curr_vec = vtype::loadu(arr + left + i * vtype::numlanes);
```

Comment:
Isn't this exactly what `partition_vec` function does? I think we should be able to use that function here in this loop.